src/include/daos/common.h

```diff
@@ -655,6 +655,8 @@ enum {
 #define DAOS_VC_LOST_REPLICA		(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x43)
 
 #define DAOS_NVME_FAULTY		(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x50)
+#define DAOS_NVME_BIO_WRITE_ERR		(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x51)
+#define DAOS_NVME_BIO_READ_ERR		(DAOS_FAIL_UNIT_TEST_GROUP_LOC | 0x52)
```

Comment:
minor: DAOS_NVME_WRITE_ERR and DAOS_NVME_READ_ERR looks shorter and better.

Reply:
I will update and remove BIO